.github/workflows/ci.yml

```
@@ -0,0 +1,14 @@
+name: Continuous Integration
+on: [pull_request]
+
+jobs:
+    ci:
+        name: Tests & Quality
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@master
+            - uses: shivammathur/setup-php@v2
+              with:
+                  php-version: 7.4
+            - run: composer install --prefer-dist --no-interaction
```

```suggestion
            - run: make install
```

except if it does not work?